language/de/acp/board.php

```diff
@@ -555,8 +555,8 @@
 $lang = array_merge($lang, array(
 	'ACP_EMAIL_SETTINGS_EXPLAIN'	=> 'Diese Informationen werden benötigt, wenn das Board E-Mails an deine Benutzer sendet. Stelle bitte sicher, dass die von dir angegebene Adresse gültig ist; geblockte oder nicht zustellbare Nachrichten werden an diese Adresse geschickt. Falls dein Webhosting-Provider keinen PHP-basierten E-Mail-Dienst anbietet, kannst du deine Nachrichten auch direkt über SMTP versenden. Dies erfordert die Angabe der Adresse eines geeigneten Servers (frage falls nötig deinen Provider). Falls der Server eine Authentifizierung erfordert (und nur, wenn dies der Fall ist), gib den Benutzernamen und das Passwort ein und wähle eine Authentifizierungsmethode aus.',
 
-	'ADMIN_EMAIL'					=> 'Antwort-E-Mail-Adresse',
-	'ADMIN_EMAIL_EXPLAIN'			=> 'Diese technische Kontakt-Adresse wird als Antwort-Adresse für alle E-Mails genommen. Sie wird in allen E-Mails als <samp>Rückleitungs</samp>- und <samp>Absender</samp>-Adresse verwendet.',
+	'ADMIN_EMAIL'					=> 'Von-E-Mail-Adresse',
```

Comment:
Irgendwie gefällt mir das nicht (was aber auch an der englischen Fassung liegen kann :-)). Zumindest im Titel würde ich "Absende-..." verwenden

Reply:
Oder besser "Absender-Adresse"? Ich bin so oder so für einheitliche Benennung in beiden  Feldern.

Reply:
Wenn wir uns an Wikipedia orientieren:
From: Absender
Reply-To: Antwortadresse
Sender: Technischer Absender